app/models/budget/phase.rb

```diff
@@ -19,9 +20,9 @@ class Phase < ApplicationRecord
 
     validates_translation :name, presence: true
     validates_translation :description, length: { maximum: DESCRIPTION_MAX_LENGTH }
+    validates_translation :main_link_url, presence: { if: -> { main_link_text.present? }}
```

Comment:
```suggestion
    validates_translation :main_link_url, presence: true, unless: -> { main_link_text.blank? }
```

For the same reasons mentioned in the comment about the budget model.